core/services/pipeline/runner.go

```diff
@@ -386,11 +371,11 @@ func (r *runner) executeTaskRun(ctx context.Context, spec Spec, taskRun *memoryT
 	taskTimeout, isSet := taskRun.task.TaskTimeout()
 	if isSet {
 		var cancel context.CancelFunc
-		ctx, cancel = utils.CombinedContext(r.chStop, taskTimeout)
+		ctx, cancel = utils.CombinedContext(ctx, r.chStop, taskTimeout)
```

Comment:
This is a mistake. We should remove this line and the change below it.

Reply:
Done